src/GregClient/Requests/PackageManagerRequest.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using RestSharp;
+using System.IO;
+
+namespace Greg.Requests
+{
+    public class PackageManagerRequest : Request
+    {
+        private Method _httpMethod;
+        public readonly bool fileRequest;
+        internal string fileToUpload;
+
+        public PackageManagerRequest(string path, RestSharp.Method httpMethod, bool fileRequest = false, string fileToUpload = "") : base()
+        {
+            _path = path;
+            _httpMethod = httpMethod;
+            this.fileRequest = fileRequest;
+            this.fileToUpload = fileToUpload;
+        }
+
+        public PackageManagerRequest(string path, RestSharp.Method httpMethod)
```

Comment:
This constructor is not required because the above constructor already has default values for other two arguments.